BatteryExamine.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+
+public class BatteryExamine
+{
+
+    string message = string.Empty;
+    Dictionary<string, string> batteryAlerts = new Dictionary<string, string>();
+
+    public Dictionary<string, string> BatteryIsOk(BatteryFactors batteryFactors)
+    {
+        batteryAlerts.Clear();
+        if (batteryFactors.temperatureMesureUnit == "Fahrenheit")
+        {
+            batteryFactors.batteryTemprature = BatteryFeatureConversion.ConvertTemperatureFahrenheitToCelsius(batteryFactors.batteryTemprature);
```

Comment:
Why BatteryIsOk should call convertion function ...then it should know existence of unit of temperature ? if we add new type of temperature unit ?